### PR TITLE
LPS-144042 Support both classPK/key identifiers in translations

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:translation

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Failure seems legit. I'll take a look at it.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ImportTranslationMVCActionCommand.java [hunk 1]
```diff
@@ -153,11 +160,11 @@ protected void doProcessAction(
 				).setParameter(
 					"classNameId", classNameId
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ImportTranslationMVCActionCommand.java:161]: Why do not use `getClassNameId` of the `translationRequestHelper` too here?

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Applied your suggestion (squashed in the last commit). If you're ok with the rest, please forward.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:translation

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1327401_1721

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by adolfopa** [comment]
Forwarded as part of https://github.com/brianchandotcom/liferay-portal/pull/111542

**Pull request closed without merging**